src/tools/segmentation/BrushTool.js

```diff
@@ -63,4 +73,22 @@ export default class BrushTool extends BaseBrushTool {
 
     external.cornerstone.updateImage(evt.detail.element);
   }
+
+  handleKeyboardCallback(evt) {
+    const { key, ctrlKey } = evt.detail;
```

Comment:
Given that we will likely add new hotkeys in the future, can you extract this into a re-usable function. Let's discuss the best approach to best do this.

Reply:
c3cc50154230ac21ba11c9f00428ad2d0a1b4897